tools/prover/src/main.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright 2024 RISC Zero, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+mod plan;
+mod task_mgr;
+mod worker;
+
+use risc0_zkvm::{ExecutorEnv, ExecutorImpl, InnerReceipt, NullSegmentRef, Receipt};
+use risc0_zkvm_methods::{FIB_ELF, FIB_ID};
+
+use self::{plan::Planner, task_mgr::TaskManager};
+
+fn main() {
+    println!("Submitting proof request...");
+
+    let mut task_manager = TaskManager::new();
+    let mut planner = Planner::default();
+
+    let iterations = 100;
+    let env = ExecutorEnv::builder()
+        .write_slice(&[iterations])
+        .segment_limit_po2(15)
```

Comment:
I'm guessing we are using a low segment size just to make more jobs to test the task manager?
```suggestion
        // Use a low segment size to generate more jobs in this example.
        .segment_limit_po2(15)
```